app_dart/lib/src/request_handlers/append_log.dart

```diff
@@ -51,6 +66,13 @@ class AppendLog extends ApiRequestHandler<Body> {
       await transaction.commit();
     });
 
+    await writeToStackdriver('${encodedOwnerKey}_${task.attempts}');
+
     return Body.empty;
   }
+
+  Future<void> writeToStackdriver(String logName) async {
+    final List<String> lines = String.fromCharCodes(requestBody).split('\n');
+    await stackdriverLogger.writeLines(logName, lines);
+  }
```

Comment:
Please file a bug and add a TODO here to keep track of the UTF16 issue.

Reply:
Added! Stackdriver seems to have this at a won't fix, but I filed the issue on our tracker so we know it's a known limitation.